app-react/src/main/kotlin/com/ustadmobile/view/ContentEntryDetailOverviewScreen.kt

```diff
@@ -0,0 +1,483 @@
+package com.ustadmobile.view
+
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.hooks.useStringsXml
+import com.ustadmobile.core.impl.locale.StringsXml
+import com.ustadmobile.core.util.UMFileUtil
+import com.ustadmobile.core.viewmodel.ContentEntryDetailOverviewUiState
+import com.ustadmobile.lib.db.entities.*
+import com.ustadmobile.mui.common.justifyContent
+import com.ustadmobile.mui.common.md
+import com.ustadmobile.mui.common.xs
+import com.ustadmobile.mui.components.UstadQuickActionButton
+import csstype.AlignItems
+import csstype.JustifyContent
+import csstype.TextAlign
+import csstype.px
+import mui.icons.material.*
+import mui.material.*
+import mui.material.List
+import mui.material.styles.TypographyVariant
+import mui.system.Container
+import mui.system.responsive
+import mui.system.sx
+import react.FC
+import react.Props
+import react.create
+import react.useState
+
+external interface ContentEntryDetailOverviewScreenProps : Props {
+
+    var uiState: ContentEntryDetailOverviewUiState
+
+    var onClickDownload: () -> Unit
+
+    var onClickOpen: () -> Unit
+
+    var onClickMarkComplete: () -> Unit
+
+    var onClickDelete: () -> Unit
+
+    var onClickManageDownload: () -> Unit
+
+    var onClickTranslation: () -> Unit
```

Comment:
This needs to take the ContentEntryRelatedEntryJoinWithLanguage as a parameter for the event - otherwise the event handler won't know what language was clicked.